learners/setup.md

```diff
@@ -2,53 +2,154 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+The vantage6 has requirements on hardware and software to run its server and node instances. As for the detailed requirements, please refer to the [vantage6 documentation](https://docs.vantage6.ai/en/main/node/requirements.html).
 
-## Data Sets
+For now, we just need to focus on the software setup.
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+## Software setup
 
-## Software Setup
+This lesson requires the following software to be installed on your system:
 
-::::::::::::::::::::::::::::::::::::::: discussion
+- Operating system: Ubuntu 18.04+ , MacOS Big Sur+, or Windows 10+
+- Docker (always latest version)
+- Conda (latest version)
+- Python
+    - Python v3.10 for vantage6 version 3.8.0 or higher
+    - Python v3.7 for other lower versions of vantage6
```

Comment:
Remove the Python 3.7 requirement, during the workshop we will work only with the newest version of vantage6.